socorro/cron/crontabber_app.py

```diff
@@ -832,13 +832,14 @@ def list_jobs(self, stream=None):
                     stream.write(' (in %s)\n' % timesince(_now, info['next_run']))
             else:
                 stream.write('none\n')
-            if info.get('last_error'):
+            if info.get('last_error') not in (None, '{}'):
```

Comment:
In the previous table, `last_error` was a JSON field. In the new table, it's a text field. So we need to do this janky thing to match the possible values where `None` is if there was no entry and `{}` is an empty `last_error` value.